testsuite/MDAnalysisTests/rng.py

```diff
@@ -0,0 +1,39 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+import numpy as np
+from numpy.testing import TestCase
+import random
+
+seed = 3856349111 # from: np.uint32(hash("MDAnalysis"))
+
+class RandomState(TestCase):
```

Comment:
@kain88-de's way only works for the simplest cases. Many tests call random numbers several times (say, to make two random matrices). If said function initializes the state every time the two matrices will always be the same. Unless you want to pass to the test the burden of specifying a new seed every time you call random.

In addition, some tests call specialized methods, such as `np.random.normal`. It isn't clear how to deal with these if all you have is a function call.

What if I change the current approach from "Is a RandomState" to a "Has a RandomState" in a way that tests have an instance of `MDAnalysisTests.rng.RandomState`? (so no need for inheriting anything, and we avoid having to call `super`).